pydicom/dataelem.py

```diff
@@ -713,6 +713,7 @@ def DataElement_from_raw(raw_data_element, encoding=None):
     if config.data_element_callback:
         data_elem = config.data_element_callback
         raw = data_elem(raw_data_element,
+                        encoding=encoding,
                         **config.data_element_callback_kwargs)
     VR = raw.VR
```

Comment:
Not related to the PR, but for clean-up while we're here, I'd like to get rid of the extra (confusing) `data_elem` assignment.  It was probably done before to avoid line-length style issues, but now can be:

```python
    if config.data_element_callback:
        raw = config.data_element_callback(
            raw_data_element,
            encoding=encoding,
            **config.data_element_callback_kwargs
        )
```